pkgs/development/tools/build-managers/goredo/default.nix

```diff
@@ -0,0 +1,56 @@
+{ buildGoModule
+, fetchurl
+, lib
+, zstd
+, sharness
+, python3
+, perl
+}:
+
+buildGoModule rec {
+  pname = "goredo";
+  version = "1.21.0";
+
+  src = fetchurl {
+    url = "http://www.goredo.cypherpunks.ru/download/${pname}-${version}.tar.zst";
```

Comment:
There https is invalid. Do we have an URL with a valid certificate?

Reply:
We have not, the author uses his own chain of trust.